### ISSUE TITLE
How to address 3 "empty" errors about an xlsx file

### ISSUE THREAD

**Issue by hepcat72** [super-linter/super-linter#4027]
I added 3 excel files to my repo and superlinter is complaining about one of them in 3 different errors (`EDITORCONFIG`, `GITLEAKS`, and `JSCPD`).  I don't understand what the errors are about because there are no error codes and no error descriptions...  How do I quiet these errors?  What is even wrong?

You can see here that I tried manually running `jscpd` on the command line on the file that superlinter said it complained about, and there is no error.  Is this a superlinter bug?

```
docker run --pull=always -e 'FILTER_REGEX_INCLUDE=(DataRepo\/example_data\/testing_data\/accucor_with_multiple_labels\/accucor_bad_label\.xlsx)' -e LINTER_RULES_PATH=/ -e IGNORE_GITIGNORED_FILES=true -e RUN_LOCAL=true -v /Users/rleach/PROJECT-local/TRACEBASE/tracebase/:/tmp/lint github/super-linter:slim-v4.9.4
slim-v4.9.4: Pulling from github/super-linter
Digest: sha256:33ceb6b4e3144aabb48a3ae9d514dc69016ad52b85012cbc3f855e4af604f03b
Status: Image is up to date for github/super-linter:slim-v4.9.4


...


2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   Linting [EDITORCONFIG] files...
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   ---------------------------
2023-03-27 22:04:31 [INFO]   File:[/tmp/lint/DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor_bad_label.xlsx]
2023-03-27 22:04:31 [ERROR]   Found errors in [editorconfig-checker] linter!
2023-03-27 22:04:31 [ERROR]   This file should have failed test case!
2023-03-27 22:04:31 [ERROR]   Error code: 0. Command output:
------

------
2023-03-27 22:04:31 [INFO]   
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   Linting [GITLEAKS] files...
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   ----------------------------------------------
2023-03-27 22:04:31 [INFO]   ---------------------------
2023-03-27 22:04:31 [INFO]   File:[/tmp/lint/DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor_bad_label.xlsx]
2023-03-27 22:04:31 [ERROR]   Found errors in [gitleaks] linter!
2023-03-27 22:04:31 [ERROR]   This file should have failed test case!
2023-03-27 22:04:31 [ERROR]   Error code: 0. Command output:
------

    ○
    │╲
    │ ○
    ○ ░
    ░    gitleaks 

10:04PM INF scan completed in 2.1806ms
10:04PM INF no leaks found
------
2023-03-27 22:04:32 [INFO]   
2023-03-27 22:04:32 [INFO]   ----------------------------------------------
2023-03-27 22:04:32 [INFO]   ----------------------------------------------
2023-03-27 22:04:32 [INFO]   Linting [JSCPD] files...
2023-03-27 22:04:32 [INFO]   ----------------------------------------------
2023-03-27 22:04:32 [INFO]   ----------------------------------------------
2023-03-27 22:04:32 [INFO]   ---------------------------
2023-03-27 22:04:32 [INFO]   File:[/tmp/lint/DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor_bad_label.xlsx]
2023-03-27 22:04:33 [ERROR]   Found errors in [jscpd] linter!
2023-03-27 22:04:33 [ERROR]   This file should have failed test case!
2023-03-27 22:04:33 [ERROR]   Error code: 0. Command output:
------
Detection time:: 3.271ms
------
2023-03-27 22:04:36 [INFO]   ----------------------------------------------
2023-03-27 22:04:36 [INFO]   ----------------------------------------------
2023-03-27 22:04:36 [INFO]   The script has completed
2023-03-27 22:04:36 [INFO]   ----------------------------------------------
2023-03-27 22:04:36 [INFO]   ----------------------------------------------
2023-03-27 22:04:36 [ERROR]   ERRORS FOUND in EDITORCONFIG:[1]
2023-03-27 22:04:36 [ERROR]   ERRORS FOUND in GITLEAKS:[1]
2023-03-27 22:04:36 [ERROR]   ERRORS FOUND in JSCPD:[1]
2023-03-27 22:04:36 [FATAL]   Exiting with errors found!
+ exit
gen-rl-macbookair[2023-03-27 18:04:37]:~/PROJECT-local/TRACEBASE/tracebase$ jscpd DataRepo/example_data/testing_data/accucor_with_multiple_labels/accucor_bad_label.xlsx
Detection time:: 0.492ms
gen-rl-macbookair[2023-03-27 18:05:38]:~/PROJECT-local/TRACEBASE/tracebase$ 
```

**Comment by hepcat72** [comment]
This appears to be related to #3949 and #1288.
